packages/terra-clinical-header/CHANGELOG.md

```diff
@@ -3,12 +3,18 @@ ChangeLog
 
 Unreleased
 ----------
-### Changed 
-* Replaced inline stylings for terra-dev-site components with .scss files
-
 ### Changed
+* Replaced inline stylings for terra-dev-site components with .scss files
 * Update wdio spec to use Terra.describeViewports
 
+### Added
+* Add new theme variables for height
```

Comment:
Should we do a pr to the dark theme and consumer theme repo before we add these?

Reply:
I'll open one for consumer. I can't speak for radiology, but judging from the pull request @bjankord opened in their space, they'd prefer to handle changes themselves.

@cwalten @PseudoFaux should I create a pull request?

Reply:
Actually, consumer doesn't theme any clinical components

Reply:
open one up for radiology, we mis-understood their pr process and they accepted the change.

Reply:
On review of the themes, it seems like we're fine to go forward without prs

Reply:
The theme does not define theme variables for css custom attributes that use default values. We'll let radiology handle this change if they want a different height.